Client/Exceptions/CompletionException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Client.Exceptions
+{
+    public class CompletionException : Exception
+    {
+        public CompletionException(string message) : base(message)
+        {
+        
```

Comment:
```suggestion
        // Intentional left empty...
```
Bare for at vise at det er med vilje